### DISCUSSION TITLE
Gesture manager: Disable touch input

### DISCUSSION THREAD

**Discussion by avalonv** [koreader/koreader#10180] [Q&A] [answered]
I recall it being possible to set a gesture to temporarily disable touch input. However, I seemingly can't find that option anywhere in the gestures manager anymore. Did I misremember or has that option been moved somewhere else?

**Comment by NiLuJe** [comment]
It's available as a dispatcher action (.e.g, a gesture manager action) in "Device settings", as "Toggle touch input".

**Reply by avalonv** [reply]
I apologise, but could you demonstrate in a step by step which menus I need to navigate through to to find that option? I can't seem to find it on my device.

**Comment by NiLuJe** [comment, marked as answer]
[Gear] > Taps and gestures > Gesture manager > [your favorite gesture here] > Device > Toggle touch input

**Reply by avalonv** [reply]
Okay, it appears I wasn't going insane. This option wasn't present on the version I was using (202208), but after updating to the latest release I can enable it.
